### DISCUSSION TITLE
Wake up panel when Google Home or Alexa palying music

### DISCUSSION THREAD

**Discussion by ady1972** [Blackymas/NSPanel_HA_Blueprint#1154] [Ideas]
I'm trying to realize an idea that i have on mind...
I would like that NSPanel wake up on a specific media player entity when some music is playing on it.
Actually i have created a button page with this entity that will be show when music start and then i need to press the button to have media player entity control on display.
I think that at now is not possible to have directly an entity page as wake up page.
I'm tryn to modify a wake up page with Nextion but without success for now (i need to understand beeter how rto use it :) ).
Thanks a lot for any eventual suggestion

**Comment by edwardtfn** [comment]
I'm writing from my phone, so I will be limited here, but you can call the service `esphome.xxxxx _open_entity_settings_page`

**Reply by ady1972** [reply]
Many thanks....i will try... :)

**Comment by Doughboy68** [comment]
It is possible to call the media player page, but I am not sure how to call a specific button page. Only service I can see is for open entity settings page. buttonpage01 . will have to check and see what actually does when I am in front of my panel, but the status shows it changes. Probably have to "make" the button page by printf all the entities etc. and not sure how to do that - It should bepossible to make custom buttons selection pages via an automation calling the appropriate services? @ady1972 how did you make the button page appear when music starts?


You would also have to set the Timeout Page to 0 if didn't want it to go back to the home page, then there is the problem of figuring out when to set it back... eg. if media player stops, but if someone in the meantime exits that page, it could get stuck on a different one.

[https://github.com/Blackymas/NSPanel_HA_Blueprint/discussions/1016](url)

Could also place the media player as a home screen button for easy access.

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/16811687/eea745b4-7a9e-48dc-a5b1-4142a5286c43)

**Comment by ady1972** [comment]
To make appear button page i have done an automation that when a particular media player entity start play music do this operation
Set as home page entity button desired
Change Timeout page, dimmer time and sleep time (to =0) 
Wake up display
And then to see entity media player i need to press corresponding button that will remain dispalyed until music is playng.
When music stop another automation will reset all value to default including home page.
I will try your suggest...Many thanks

**Reply by edwardtfn** [reply]
Try this:
```yaml
service: esphome.xxxxx_panel_send_command_printf
data:
  cmd: page buttonpage02
```

**Reply by ady1972** [reply]
It works like a charm using command esphome.xxxxx _open_entity_settings_page. Many thanks now i am verry happy :) :)
